rust/gui-client/src-common/src/deep_link.rs

```diff
@@ -133,7 +133,7 @@ mod tests {
         Ok(())
     }
 
-    fn parse_callback_wrapper(s: &str) -> Result<super::AuthResponse> {
+    fn parse_callback_wrapper(s: &str) -> Result<crate::auth::Response> {
```

Comment:
```suggestion
    fn parse_callback_wrapper(s: &str) -> Result<auth::Response> {
```

Seems to already be in scope.